applications/IgaApplication/custom_processes/snake_sbm_process.cpp

```diff
@@ -115,7 +115,7 @@ void SnakeSbmProcess::CreateTheSnakeCoordinates(
     mesh_sizes_uv[0] = knot_step_uv[0]; 
     mesh_sizes_uv[1] = knot_step_uv[1];
     auto& surrogate_model_part = rIgaModelPart.GetSubModelPart(surrogate_sub_model_part_name);
-    surrogate_model_part.GetProcessInfo().SetValue(KNOT_SPAN_SIZES, mesh_sizes_uv);
+    surrogate_model_part.GetParentModelPart().SetValue(KNOT_SPAN_SIZES, mesh_sizes_uv);
```

Comment:
I remember discussing for a while about this until we got to this solution, which makes very much sense in this case.

Having said this, I'd put a comment like "Note that in here we are saving the knot span info in the parent model part database" to highlight what we are doing (it is not very customary).